extensions/asde-deployment/package.json

```diff
@@ -28,6 +28,7 @@
           "light": "./images/sqldb_edge.svg",
           "dark": "./images/sqldb_edge_inverse.svg"
         },
+        "tags": ["Hybrid", "SQL Server"],
```

Comment:
I think we should highlight the IoT Edge here as well, please work with PM to confirm.